raiden-network/light-client
# Sort token list by balance and symbol

**Thank you for submitting this pull request :)**

Resolves #1424 

**Short description**


**Definition of Done**


**Steps to manually test the change (dApp)**

1. Serve dApp
2. Connect
3. Click on token on transfer screen
4. Click connect new token
5. Check sorting in the token list
